### PR TITLE
feat: installer configures weeklynet

### PR DESCRIPTION
**Pull request by johnyob**
# Context

# Description

Adds weeklynet network config to the installer script

# Manually testing the PR

Having deleted / moved your `~/.jstz/config.json`, run:
```
source scripts/install-jstz-cli.sh
```

### CODE REVIEW

**Review by ryutamago — CHANGES_REQUESTED** [review]
shouldn't we check for the full path? 
```
if [ -d "$jstz_home/config.json" ]; then
        echo "Configuration already exists. Skipping..."
```

File: scripts/install-jstz-cli.sh
Comment (alanmarkoTrilitech) [inline scripts/install-jstz-cli.sh:51]: where is the JSTZ_VERSION defined?

Comment (alanmarkoTrilitech) [inline scripts/install-jstz-cli.sh:51]: The manual alias and shell alias differ

**Review by alanmarkoTrilitech — CHANGES_REQUESTED** [review]
The network change seems good.

Added some comments regarding other changes in the install file as I couldn't do it before (please next time don't merge the review commits automatically).

I think it would be nice if the install script also checked the npm, and encouraged the user to install it if it isnt + also if it automatically cloned the examples folder that we currently have in the quick start guide(I don't think there is a need to clone the whole repo, just the folder).  

I've also had an issue with having old alias in .zshrc - not sure if it was because of JSTZ_VERSION being wrong or just having old version of it, but I guess it is fine for the first time user.

File: scripts/install-jstz-cli.sh
Comment (johnyob) [reply scripts/install-jstz-cli.sh:51]: It was moved to the `$version` variable -- it is not standard for all bash variables to be caps (just non-local variables)

Comment (johnyob) [reply scripts/install-jstz-cli.sh:51]: Will fix now

**Comment by johnyob** [comment]
> shouldn't we check for the full path?

What do you mean by the 'full path'?

**Comment by johnyob** [comment]
> I think it would be nice if the install script also checked the npm, and encouraged the user to install it if it isnt + also if it automatically cloned the examples folder that we currently have in the quick start guide(I don't think there is a need to clone the whole repo, just the folder).

I disagree. This is not the responsibility of the installer, but likely of a `template` command in the CLI

File: scripts/install-jstz-cli.sh
Comment (johnyob) [reply scripts/install-jstz-cli.sh:51]: Fixed 9169132759c687f3ca26de7139ffdc820c3ea119

**Comment by ryutamago** [comment]
> > shouldn't we check for the full path?
> 
> What do you mean by the 'full path'?

i meant instead of checking if `.jstz` exists, we should check if `.jstz/config.json`exists.

**Review by alanmarkoTrilitech — APPROVED** [review]
:ship:

**Review by ryutamago — APPROVED** [review]
🔥

**Pull request merged**
